### ISSUE TITLE
[sugfestion] config for recipe transfer

### ISSUE THREAD

**Issue by Biviho** [Peatral/EmbersConstruct#13]
hi! would it be possible to add a config option to decide witch recipe group to transfer?

for example if i want the melter recipe to not be added but instead the stamper to be added from tinker's recipes

this way a modpack maker can have more control to witch recipe are added

**Comment by Peatral** [comment]
This is actually a good idea. Currently I can't work on the mod but next week it should be implemented, it is quite easy.

**Comment by Biviho** [comment]
yeah, i've encountered this while working on my pack, even better would be to been able to pick/lacklist specific recipes (for example by output) but i guess this is way more complicated

btw, ty for the nice mod!

**Comment by Peatral** [comment]
making the mod completely adjustabpe. would be nice but currently not in my capacities, although I am trying to make everyone happy.

**Comment by Biviho** [comment]
don't worry, the fact that you made a mod to connect tinker and emberis already great!
